The Ugly Indian Hackathon

harshabhat86 opened the pull request:
Please find the submission by @harshabhat86 and @sharubhat
Access the working copy in this url harshabhat86.github.io/TUI/index.html

Thanks.

harshabhat86 committed:
The Ugly Indian Hackathon

harshabhat86 committed:
The missing i added

harshabhat86 committed:
The README shared over email has been checked in

indiarising commented:
Hello,

We are overwhelmed by the effort put in by you! We have received a lot more submissions than we anticipated and the quality of work has been amazing.

We understand that some of you found out about this a bit late and haven't had the time to finish everything. You can still patch up your github submissions and documentation. We are going to share the work with the community and the easier you make it for people to just browse through your work, the better it is for everyone.

We will get back to you within a few days and evaluate how to take this further.

As a concluding note for tonight, all we would say is that your work proves that India is Rising!

Thanks again!

Note - Some of you might have also received this as an email. Somehow github doesn't let us know your email at times during pull requests. So letting you know via this comment.

harshabhat86 committed:
Maps integrated to create Spotfix

1) Click on map to get the location
2) Enter location to find the spot on map
3) Create spot fix.

harshabhat86 committed:
Date format bug fix
